### ISSUE TITLE
Add support for importing psafe3 files

### ISSUE THREAD

**Issue by donalhunt** [buttercup/buttercup-importer#56]
Password Safe (https://pwsafe.org/) uses the psafe3 fle format. It would be great to add support for importing from that file format.

Workaround in the meantime seems to be:
1. Covert to keepass format using something like https://www.flyingpenguin.com/?p=23650
2. Import the keepass file into Buttercup.

... or just export the psafe3 file to csv and import that.

**Comment by perry-mitchell** [comment]
Good idea! Some investigation would need to be done on the format to make sure it's practical to import. Importers can't include any binaries for non-node programs as bundling them in the desktop application is nigh on impossible. If it's entirely JS it should be fine.

**Comment by donalhunt** [comment]
The psafe3 format is documented at https://www.github.com/dol/node-passwordsafe/tree/master/doc%2FformatV3.txt

A node JS library is available in the same repository.
